### ISSUE TITLE
Prediction sample gain

### ISSUE THREAD

**Issue by tonegas** [tonegas/nnodely#24]
Adding a gain along the window for recurrent train, here:

            ## Calculate the total loss
            total_loss = 0
            for ind in range(len(self.minimize_dict)):
                total_loss += sum(horizon_losses[ind])/(prediction_samples+1)
                aux_losses[ind][idx//batch_size] = total_loss.item()
